### PR TITLE
Enabled xml doc generation

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@NTaylorMullen

### CODE REVIEW

File: src/Microsoft.AspNetCore.Testing/ReplaceCulture.cs [hunk 1]
```diff
@@ -45,11 +46,27 @@ public ReplaceCultureAttribute(string currentCulture, string currentUICulture)
         /// want to be able to find bugs where we're accidentally relying on the Invariant instead of the
         /// user's culture.
         /// </remarks>
+#else
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Testing/ReplaceCulture.cs]: Could just ifdef/else the summary.

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Testing/ReplaceCulture.cs]: It is not letting me insert ifdefs in the middle of doc comments.

Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Testing/ReplaceCulture.cs]: Ah kk

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**